app/models/user/tasks/checklist_item.rb

```diff
@@ -4,7 +4,8 @@ class ChecklistItem
       DISPLAY_CATEGORIES = {
         "ADMA" => "newStudent",
         "ADMP" => "admissions",
-        "FINA" => "financialAid"
+        "FINA" => "financialAid",
+        "SFAC" => "financialAid"
```

Comment:
(commentary:  we should turn "trailing commas" on in our Prettier settings so line 7 wouldn't show up as a new diff)